src/core/get-opened-at.ts

```diff
@@ -0,0 +1,20 @@
+import { DateTime } from 'luxon';
+import { Event, PullRequest } from '../models/models';
+
+export const getOpenedAt = (
```

Comment:
the ttl will be based on this datetime. It's useful because sometimes we create a PR but it's in draft several days until it's ready so we will find the newest "reopened" event to get that datetime